CDCgov/prime-reportstream
# Add codeowners for backend

This PR adds the newly created GitHub CDC team [PRIME-ReportStream-CODEOWNERS-backend](https://github.com/orgs/CDCgov/teams/PRIME-ReportStream-CODEOWNERS-backend) as CODEOWNERS for any file in the 
`/prime-router` folder.

Test Steps:
1. Go to "Files changed" and make sure the CODEOWNERS automatic checker shows no errors. Prior to setting up the codeowners team correctly, the checker was showing this:

![Screenshot 2023-09-27 at 3 03 03 PM](https://github.com/CDCgov/prime-reportstream/assets/118766341/62b425d7-76e3-42b8-a29b-d20d68a9ba45)

2. Verify that CODEOWNERS review required is enabled on this repository
